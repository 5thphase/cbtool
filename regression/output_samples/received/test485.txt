 status: Waiting for vm_4 (cloud-assigned uuid E91C42CA-029C-5AE8-95D4-BDAE8A50C6E7) to start...
 status: Trying to establish network connectivity to vm_4 (cloud-assigned uuid E91C42CA-029C-5AE8-95D4-BDAE8A50C6E7), on IP address 142.190.173.104...
 status: Bypassing the sending of a copy of the code tree to vm_4 (142.190.173.104)...
 status: Bypassing generic VM post_boot configuration on vm_4 (142.190.173.104)...
VM object 9B1B97E3-95E9-5B28-960B-4F71F9119D35 (named "vm_4") sucessfully attached to this experiment. It is ssh-accessible at the IP address 142.190.173.104 (E91C42CA-029C-5AE8-95D4-BDAE8A50C6E7.simcloud.com).
The following ATTACHED VMs are in this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |uuid
|vm_1          |tinyvm              |pico32      |238.219.171.43  |simhostc1                  |SUT                 |none    |none       |5C3901F8-BB8C-5D41-AFA5-8507C3FB1C4A
|vm_2          |tinyvm              |pico32      |191.64.121.3    |simhostd1                  |LG                  |none    |none       |DE14ABC9-4F02-500B-8E42-11D6A0678ADA
|vm_3          |tinyvm              |pico32      |95.156.75.163   |simhostb1                  |SUT                 |none    |none       |764DCC85-E8B4-535C-892F-21EAA7EB7475
|vm_4          |db2                 |copper32    |142.190.173.104 |simhosta1                  |SUT                 |none    |none       |9B1B97E3-95E9-5B28-960B-4F71F9119D35

The VM object vm_4, attached to this experiment, has the following attributes (Cloud TESTCLOUD) :
|attribute (VM object key)               |value                                
|host_name                               |simhosta1                            
|meta_tags                               |{'a': 'b', 'x': 'y', 'r': '2'}       
|size                                    |copper32                             
|vmc_name                                |simzone_a                            
|vmc_pool                                |SUT                                  

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |1.66M                         
|Redis Version                                        |2.6.16                        
|Uptime (in seconds)                                  |1793002                       
|Total Connections Received                           |5116                          
|Total Commands Processed                             |812063                        
|Number of Keys                                       |328                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.4.6                         
|Storage Size                                         |1511424                       
|Data Size                                            |448464                        
|Index Size                                           |376096                        
|Average Object Size                                  |512.530285714                 
|Collections                                          |14                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |4                             
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |4                             
|VMs ARRIVED                                          |4                             
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |0                             
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |0                             
|AI RESERVATIONS                                      |0                             
|AIs ARRIVED                                          |0                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |92                            

 ---------------- VMC-WIDE COUNTERS ----------------
|C3B836AF-6AAE-5401-84E8-B00701C39D77 (simzone_d) VM RESERVATIONS|1                             
|2F03C41C-37EA-529D-873F-A59AD9D097ED (simzone_b) VM RESERVATIONS|1                             
|85B1573F-AFCB-5D91-861C-A9290E0BC814 (simzone_a) VM RESERVATIONS|1                             
|9EA9AC96-B5DE-50B4-93B7-F0933B0AEB65 (simzone_c) VM RESERVATIONS|1                             

