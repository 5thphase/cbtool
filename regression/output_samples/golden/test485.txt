 status: Waiting for vm_4 (cloud-assigned uuid A5F36267-AA9B-5012-815F-5E9F90767572) to start...
 status: Trying to establish network connectivity to vm_4 (cloud-assigned uuid A5F36267-AA9B-5012-815F-5E9F90767572), on IP address 67.13.107.208...
 status: Bypassing the sending of a copy of the code tree to vm_4 (67.13.107.208)...
 status: Bypassing generic VM post_boot configuration on vm_4 (67.13.107.208)...
VM object 0D535CBB-6E66-5F28-BDAE-28CF512F3907 (named "vm_4") sucessfully attached to this experiment. It is ssh-accessible at the IP address 67.13.107.208 (A5F36267-AA9B-5012-815F-5E9F90767572.simcloud.com).
The following ATTACHED VMs are in this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |uuid
|vm_1          |tinyvm              |pico32      |236.168.198.133 |simhostb1                  |SUT                 |none    |none       |F2DE6C84-C41F-54D4-A409-1BE7003DCF5D
|vm_2          |tinyvm              |pico32      |191.55.71.199   |simhostd1                  |LG                  |none    |none       |27441C86-DA57-531E-B5F3-606F34C0EC91
|vm_3          |tinyvm              |pico32      |70.181.54.169   |simhostb1                  |SUT                 |none    |none       |06DDC610-2A19-59CD-91EE-8FE70AA0F412
|vm_4          |db2                 |copper32    |67.13.107.208   |simhostc1                  |SUT                 |none    |none       |0D535CBB-6E66-5F28-BDAE-28CF512F3907

The VM object vm_4, attached to this experiment, has the following attributes (Cloud TESTCLOUD) :
|attribute (VM object key)               |value                                
|host_name                               |simhostc1                            
|meta_tags                               |{'a': 'b', 'x': 'y', 'r': '2'}       
|size                                    |copper32                             
|vmc_name                                |simzone_c                            
|vmc_pool                                |SUT                                  

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |2.33M                         
|Redis Version                                        |2.5.12                        
|Uptime (in seconds)                                  |2417290                       
|Total Connections Received                           |4545                          
|Total Commands Processed                             |3888871                       
|Number of Keys                                       |326                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.4.5                         
|Storage Size                                         |5448777728.0                  
|Data Size                                            |4961947736.0                  
|Index Size                                           |133726656                     
|Average Object Size                                  |8103.99706019                 
|Collections                                          |40                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |4                             
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |4                             
|VMs ARRIVED                                          |4                             
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |0                             
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |0                             
|AI RESERVATIONS                                      |0                             
|AIs ARRIVED                                          |0                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |92                            

 ---------------- VMC-WIDE COUNTERS ----------------
|CEA17C37-820A-507F-9356-90956267A568 (simzone_a) VM RESERVATIONS|0                             
|966D86CB-90BB-553D-A06A-0FFA0818D568 (simzone_d) VM RESERVATIONS|1                             
|C41BC369-F411-5382-B95F-70C31BD20E23 (simzone_c) VM RESERVATIONS|1                             
|2FF9728E-95B3-5EE9-A0A4-61875B507FA8 (simzone_b) VM RESERVATIONS|2                             

